scripts/nf1_classifier.py

```diff
@@ -361,8 +366,10 @@ def plot_decision_function(score_df, partition, output_file):
                             y_score = clf.decision_function(x_test)
                             y_score_train = clf.decision_function(x_train)
 
-                            fpr, tpr, _ = roc_curve(y_test, y_score)
-                            fpr_t, tpr_t, _ = roc_curve(y_train, y_score_train)
+                            fpr, tpr, _ = roc_curve(y_test, y_score,
```

Comment:
See [`numpy.interp`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.interp.html) and [this example](http://scikit-learn.org/stable/auto_examples/model_selection/plot_roc.html). You will want to choose say 1000 FPRs from 0 to 1 and then get the corresponding TRP for each ROC curve. Then it will be a simple averaging of TPRs for each FPR and you will have your beautiful plot.